### PR TITLE
Capstone Project

### PR DESCRIPTION
**Pull request by ibr5500**
## in this project:
- I have made a capstone to what we have learned past weeks.
- Created two pages (home and about) and styled them.
- Created mobile menu.
- Dynamically created featured speakers cards by JavaScript.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,73 @@
...
+- Html
+- Css
+- JavaScript
+- VS Studio
+- Git Bash
+
+## Live Demo (if available)
+
+[Click](https://ibr5500.github.io/Capstone/) for Live Demo Link
+
```
Comment (phelian23) [inline README.md]: - [ ] Kindly record a video presenting the features of the project you built. It is a video with a maximum length of 5 minutes. The content of the video should include:

- A description of the project.
- A demo of the project features (different pages and different screen sizes).
- You should also highlight some interesting piece of code or something you built that you are very proud of.

File: README.md [hunk 2]
```diff
@@ -1 +1,73 @@
...
+Please give a ⭐️ if you like this project!
+
+
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration 
+- etc
+
```
Comment (phelian23) [inline README.md]: - [ ] Kindly give appropriate credit to the author [Cindy Shin](https://www.behance.net/adagio07) in the README of your project.

File: index.js [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+  document.querySelector('body').style = 'overflow: hidden';
+  document.querySelector('.campaign').style = 'display: none';
+});
+
+closeButton.addEventListener('click', () => {
+  mobileMenu.style = 'display: none';
+  hamburger.style = 'display: block';
+  document.querySelector('body').style = 'overflow: scroll';
+  document.querySelector('.campaign').style = 'display: flex';
+  window.location.reload();
```
Comment (phelian23) [inline index.js]: - [ ] When one closes the mobile menu, the page shouldn't reload as this is an expensive process. Kindly remove the reload option.

File: about.html [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+                <div class="about-text-container">
+                    <p class="main-text">
+                        A startup created in 2022 provides spare parts for all models of cars.<br/>Moreover, We deliver our products to you wherever you are and whenever you want.
+                    </p>
+                </div>
+                <div class="contact-text">
+                    <p>
+                        You are per E-mail away to get our services <br/> Please do not hesitate to contact us.! <br /><span>creeds@gmail.com</span>
+                    </p>
+                </div>
```
Comment (phelian23) [inline about.html]: - [ ] On the about page, the title section should be centered. kindly make the text alignment center.

**Review by phelian23 — CHANGES_REQUESTED** [review]
Hi @ibr5500, 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!


## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Comment by ibr5500** [comment]
Hello, Mr. @phelian23 I have changed all the required changes.
Thank you for the review.

**Review by MeqdamAlqudah — APPROVED** [review]
Hi @ibr5500 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ibr5500** [comment]
Hi @MeqdamAlqudah, 
Thank you for the code review.

**Pull request merged**